app/models/miq_vim_broker_worker/runner.rb

```diff
@@ -105,8 +105,7 @@ def ems_ids_for_notify(address, userid)
       ems = ManageIQ::Providers::Vmware::InfraManager
             .includes(:authentications)
```

Comment:
Probably want to changes this to `.includes(:authentications, :endpoints)` to avoid the N+1 in the `.detect` below.  Another option is `.includes(:authentications, :hostname)` if that makes it clearer.